.circleci/config.yml

```diff
@@ -253,11 +282,17 @@ jobs:
             - dep-cache-{{ checksum "Gopkg.toml" }}-{{ checksum "WORKSPACE" }}
       - run:
           command: |
-            make localTestEnv go-test KUBECONFIG=/go/src/istio.io/istio/.circleci/config
+            mkdir -p /go/out/tests
+            go get github.com/jstemmer/go-junit-report
+            trap "go-junit-report </go/out/tests/go-test-report.out > /go/out/tests/go-test-report.xml" EXIT
+            make localTestEnv go-test KUBECONFIG=/go/src/istio.io/istio/.circleci/config T=-v | tee /go/out/tests/go-test-report.out
       - store_artifacts:
           path: /go/bin
+          path: /go/out
+      - store_test_results:
+          path: /go/out/tests
```

Comment:
This is cool.. does it show stuff in a better UI?

Reply:
Example:
https://circleci.com/gh/costinm/istio/534

If a test fails it shows the failed test, and I think it has some graphs.